internal/sdks/sdk_instructions/android.md

```diff
@@ -0,0 +1,124 @@
+# Installation steps
+If you want to skip ahead, the final code is available in our [GitHub repository](https://github.com/launchdarkly/hello-android).
```

Comment:
commented this on the last PR as well but I think we'll probably want to pull this out and add to the top of the View for the show sdk step instead of having them each in here.

Reply:
Yeah, that makes sense.